### PR TITLE
Make Overlay contentReady firing after rendering content (T690303)

### PR DESCRIPTION
**Pull request by Yarovaya**

### CODE REVIEW

**Review by vconst — DISMISSED** [review]

**Review by lukyanovas — DISMISSED** [review]
LGTM

File: testing/tests/DevExpress.ui.widgets/drawer.tests.js [hunk 1]
```diff
@@ -537,6 +537,66 @@ QUnit.test("drawer panel should have correct width when async template is used",
...
+                        setTimeout(() => {
+                            $div.css("height", 600);
+                            $div.css("width", 200);
+                            args.onRendered();
+                        }, 100);
+                    }
+                }
+            }
+        }
+    }).dxDrawer("instance");
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/drawer.tests.js]: It looks like `.dxDrawer("instance")` is unnecessary.

File: testing/tests/DevExpress.ui.widgets/drawer.tests.js [hunk 2]
```diff
@@ -537,6 +537,66 @@ QUnit.test("drawer panel should have correct width when async template is used",
...
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/drawer.tests.js]: `.dxDrawer("instance")` is unnecessary.

File: testing/tests/DevExpress.ui.widgets/overlay.tests.js [hunk 3]
```diff
@@ -1412,6 +1412,36 @@ QUnit.test("behavior if option set to false", function(assert) {
...
+                            $div.css("height", 600);
+                            $div.css("width", 200);
+                            contentIsRendered = true;
+                            args.onRendered();
+                        }, 100);
+                    }
+                }
+            }
+        }
+    }).dxOverlay("instance");
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/overlay.tests.js]: `.dxOverlay("instance")` is unnecessary.

File: testing/tests/DevExpress.ui.widgets/overlay.tests.js [hunk 4]
```diff
@@ -1412,6 +1412,36 @@ QUnit.test("behavior if option set to false", function(assert) {
...
+        onContentReady: function() {
+            assert.ok(contentIsRendered, "Content is rendered before content ready firing");
+        },
+        integrationOptions: {
+            templates: {
+                "content": {
+                    render: function(args) {
+                        var $div = $("<div/>").appendTo(args.container);
+                        setTimeout(() => {
+                            $div.css("height", 600);
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/overlay.tests.js]: Are these acts required for this test case?

```
    $div.css("height", 600);
    $div.css("width", 200);
```

**Review by San4es — APPROVED** [review]
LGTM

**Pull request merged**